lib/components/primitive-components/Port/Port.ts

```diff
@@ -404,7 +404,8 @@ export class Port extends PrimitiveComponent<typeof portProps> {
       bestDisplayPinLabel = labelHints[0]
     }
 
-    const schematic_port = db.schematic_port.insert({
+    const pinAttributes = (this.parent as any)?.props?.pinAttributes
+    const schematicPortInsertProps: any = {
```

Comment:
```suggestion
    const schematicPortInsertProps: SchematicPort = {
```

idk why people are using any so much, it degrades the codebase